serenity/cluster_utils.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef SERENITY_CLUSTER_UTILS_HPP
+#define SERENITY_CLUSTER_UTILS_HPP
+
+#include <string>
+
+#include "curl_easy.h"  // NOLINT [build/include]
+
+#include "glog/logging.h"
+
+#include "rapidjson/document.h"
+
+#include "stout/error.hpp"
+#include "stout/result.hpp"
+
+/**
+ * Returns IP of host registered in Mesos  DNS hosts/
```

Comment:
Why require the knowledge of mesos-dns? You should just be able to use 'monitoring-influxdb.marathon.mesos'?